sys/newlib/Makefile.include

```diff
@@ -30,4 +30,11 @@ NEWLIB_INCLUDES ?= \
         $(foreach pat, $(NEWLIB_INCLUDE_PATTERNS), $(wildcard $(pat))), \
         -isystem $(dir))
 
-export INCLUDES += $(NEWLIB_INCLUDES)
+# If nothing was found we will try to fall back to searching for a cross-gcc in
+# the current PATH and use a relative path for the includes
+ifeq (,$(NEWLIB_INCLUDES))
+  NEWLIB_INCLUDES := $(addprefix -isystem ,$(wildcard $(dir $(shell which $(PREFIX)gcc))../$(TARGET_TRIPLE)/include))
```

Comment:
arm-none-eabi for cortex m, msp430-elf for msp430, etc.
On Mar 13, 2016 1:31 PM, "Martine Lenders" <notifications@github.com> wrote:

> In sys/newlib/Makefile.include
> <https://github.com/RIOT-OS/RIOT/pull/5053#discussion_r55936446>:
>
> > @@ -30,4 +30,11 @@ NEWLIB_INCLUDES ?= \
> >          $(foreach pat, $(NEWLIB_INCLUDE_PATTERNS), $(wildcard $(pat))), \
> >          -isystem $(dir))
> >
> > -export INCLUDES += $(NEWLIB_INCLUDES)
> > +# If nothing was found we will try to fall back to searching for a cross-gcc in
> > +# the current PATH and use a relative path for the includes
> > +ifeq (,$(NEWLIB_INCLUDES))
> > +  NEWLIB_INCLUDES := $(addprefix -isystem ,$(wildcard $(dir $(shell which $(PREFIX)gcc))../$(TARGET_TRIPLE)/include))
>
> What does TARGET_TRIPLE contain?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/5053/files#r55936446>.
>